### PR TITLE
Explore logs css refactor

### PR DESCRIPTION
**Pull request by torkelo**
when trying to style the log rows a bit I started doing some refactoring as well. But want to keep this refactoring separate from any style changes, so if the style changes are not working the refactoring can at least be used :) 

* This change moves the css classes related to the logs explore "panel" into it's own file, and renames the classes a bit to make them more unique so the nexting under ".explore" is no longer needed. 
* Changed to table instead of grid, did this only through css so we can change back more easily if we see a big issue with this. 
* added word-break rule to #14345 is fixed.

### CODE REVIEW

**Comment by torkelo** [comment]
main problem is the staged rendering causing an update to the column widths after the second pass. need to figure out a solution for that.

**Comment by torkelo** [comment]
ah, the second stage rendering column width change was present in the grid version as well.

**Review by davkal — CHANGES_REQUESTED** [review]
Rendering still seems fast enough. The level column needs fixing. And I highly recommend keeping the monospaced font for line-to-line comparisons.

**Comment by davkal** [comment]
Also fixes #14344

File: public/sass/components/_panel_logs.scss [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  // compensate for the labels padding
+  position: relative;
+  top: 4px;
+}
+
+.logs-rows {
+  // display: grid;
+  // grid-column-gap: 1rem;
+  // grid-row-gap: 0.1rem;
+  // font-family: $font-family-monospace;
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/sass/components/_panel_logs.scss]: good catch, commented that out by mistake.

File: public/sass/components/_panel_logs.scss [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  border-bottom: 1px solid $typeahead-selected-color;
+  background-color: rgba($typeahead-selected-color, 0.1);
+
+  &--preview {
+    background-color: rgba($typeahead-selected-color, 0.2);
+    border-bottom-style: dotted;
+  }
+}
+
+.logs-row__level {
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/sass/components/_panel_logs.scss]: yes! also fixed width of message when labels are hidden.

**Comment by davkal** [comment]
Looks good on the whole, but sometimes getting line break artefacts:

screenshot 2018-12-06 at 13 28 58

Unable to find a solid repro, so maybe we merge now anyway.

**Comment by davkal** [comment]
One more difference: the log levels are now butted against each other. Before, there was a row gap. 
The gap between log levels supported the visual integrity of a line, perhaps worth bringing back.

**Comment by torkelo** [comment]
Can fix the level line gap later, want to change how we show level top something that looks clearer and better.

**Pull request merged**